doc/source/flexibility/extended_mysqlbinlog.rst

```diff
@@ -4,30 +4,14 @@
 Extended ``mysqlbinlog``
 ========================
 
-|Percona Server| has implemented compression support for
-:command:`mysqlbinlog`. This is similar to support that both ``mysql`` and
-``mysqldump`` programs include (the ``-C``, ``--compress`` options "Use
-compression in server/client protocol"). Using the compressed protocol helps
-reduce the bandwidth use and speed up transfers. 
-
-|Percona Server| has also implemented support for ``SSL``.
-:command:`mysqlbinlog` now accepts the ``SSL`` connection options as all the
-other client programs. This feature can be useful with
-``--read-from-remote-server`` option. Following ``SSL`` options are now
-available:
-
- * :option:`--ssl` - Enable SSL for connection (automatically enabled with
-   other flags).
- * :option:`--ssl-ca=name` - CA file in PEM format (check OpenSSL docs,
-   implies --ssl).
- * :option:`--ssl-capath=name` - CA directory (check OpenSSL docs, implies
-   --ssl).
- * :option:`--ssl-cert=name` - X509 cert in PEM format (implies --ssl).
- * :option:`--ssl-cipher=name` - SSL cipher to use (implies --ssl).
- * :option:`--ssl-key=name` - X509 key in PEM format (implies --ssl).
- * :option:`--ssl-verify-server-cert` - Verify server's "Common Name" in its
-   cert against hostname used when connecting. This option is disabled by
-   default.
+|Percona Server| has implemented compression support for :command:`mysqlbinlog`. This option is supported in both of the ``mysql`` and ``mysqldump`` programs. The compression option helps reduce the bandwidth use and improves the transfer speed.
+
+In the `mysqlbinlog utility <https://dev.mysql.com/doc/refman/5.7/en/mysqlbinlog.html>`__   use either the ``-C`` or ``--compress`` option.
```

Comment:
The option is the same for the other programs too, and this implies that it's specific to this command.

Reply:
Rewrote the paragraphs and removed references to other programs.